astronomer/providers/microsoft/cloud/triggers/wasb.py

```diff
@@ -0,0 +1,85 @@
+import asyncio
+import logging
+from typing import Any, Dict, Tuple
+
+from aiohttp import ClientSession as Session
+from airflow.triggers.base import BaseTrigger, TriggerEvent
+
+from astronomer.providers.microsoft.cloud.hooks.wasb import WasbHookAsync
+
+log = logging.getLogger(__name__)
+
+
+class WasbBlobTrigger(BaseTrigger):
+    """
+    A trigger that fires and it finds the requested file or folder present in the given bucket.
+    :param container_name: the container in the azure cloud storage where the blobs are residing.
+    :param blob_name: the file or folder present in the container
+    :param wasb_conn_id: reference to the Azure Connection
+    :param polling_period_seconds: polling period in seconds to check for file/folder
+    """
+
+    def __init__(
+        self,
+        container_name: str,
+        blob_name: str,
+        polling_period_seconds: float,
+        wasb_conn_id: str
+    ):
+        super().__init__()
```

Comment:
`super().__init__()` can be removed as we are not using any of the `BaseTrigger` methods.